controller/workitem_blackbox_test.go

```diff
@@ -559,6 +560,65 @@ func (s *WorkItemSuite) TestUpdateWorkitemWithoutReorder() {
 	assert.Equal(s.T(), wi.Data.Attributes[workitem.SystemOrder], updated.Data.Attributes[workitem.SystemOrder])
 }
 
+// TestCreateWorkitemWithActionRule tests that when workitem is created, a defined action rule executes.
+func (s *WorkItemSuite) TestCreateWorkitemWithActionRule() {
+	// create a wit with a set transrule setting. We're using the FieldSet rule here as it it easy to test.
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItemTypes(1, func(fxt *tf.TestFixture, idx int) error {
+		fxt.WorkItemTypes[idx].TransRuleKey = rules.ActionKeyFieldSet
+		fxt.WorkItemTypes[idx].TransRuleArgument = "{ \"system.state\": \"resolved\" }"
```

Comment:
Please use backticks when you have quoted text to avoid the escaped quotes:

```go
fxt.WorkItemTypes[idx].TransRuleArgument = `{"system.state": "resolved" }`
```

Reply:
Fixed. But just to nitpick, a lot of other code, even in the same file is using the same quoting. I'd go for consistency rather than convention, but that is just my opinion.

Reply:
If consistency in a file is unreadable change it for the better :)